src/main/org/firebirdsql/gds/ng/wire/version13/V13Statement.java

```diff
@@ -0,0 +1,255 @@
+/*
+ * $Id$
+ *
+ * Firebird Open Source JavaEE Connector - JDBC Driver
+ *
+ * Distributable under LGPL license.
+ * You may obtain a copy of the License at http://www.gnu.org/copyleft/lgpl.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * LGPL License for more details.
+ *
+ * This file was created by members of the firebird development team.
+ * All individual contributions remain the Copyright (C) of those
+ * individuals.  Contributors to this file are either listed here or
+ * can be obtained from a source control history command.
+ *
+ * All rights reserved.
+ */
+package org.firebirdsql.gds.ng.wire.version13;
+
+import org.firebirdsql.gds.ISCConstants;
+import org.firebirdsql.gds.impl.wire.WireProtocolConstants;
+import org.firebirdsql.gds.impl.wire.XdrInputStream;
+import org.firebirdsql.gds.impl.wire.XdrOutputStream;
+import org.firebirdsql.gds.ng.*;
+import org.firebirdsql.gds.ng.fields.*;
+import org.firebirdsql.gds.ng.wire.FbWireDatabase;
+import org.firebirdsql.gds.ng.wire.version12.V12Statement;
+
+import java.math.BigInteger;
+import java.io.IOException;
+import java.sql.SQLException;
+
+import static org.firebirdsql.gds.ng.TransactionHelper.checkTransactionActive;
+
+/**
+ * @author <a href="mailto:mrotteveel@users.sourceforge.net">Mark Rotteveel</a>
+ */
+public class V13Statement extends V12Statement {
+
+    /**
+     * Creates a new instance of V13Statement for the specified database.
+     *
+     * @param database
+     *         FbWireDatabase implementation
+     */
+    public V13Statement(FbWireDatabase database) {
+        super(database);
+    }
+
+    @Override
+    public void execute(final RowValue parameters) throws SQLException {
```

Comment:
Method is identical to the one in V12Statement and doesn't need to be overridden here